packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -22,16 +23,66 @@ export default class CardAdoptionChain extends Component<Signature> {
       .card-adoption-chain {
         background-color: var(--boxel-200);
       }
+      .inherits-from {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding: var(--boxel-sp) 0;
+        gap: var(--boxel-sp-xxxs);
+        font: var(--boxel-font-size-sm);
+
+        position: relative;
+      }
+
+      .chain:last-child .inherits-from {
+        display: none;
+      }
+
+      .inherits-icon {
+        height: 24px;
+      }
+
+      .left-line {
+        width: 50%;
+        height: 1px;
+        background-color: var(--boxel-purple-200);
+        position: absolute;
+        top: 50%;
+        left: 0;
+        transform: translateX(-75px);
+      }
+
+      .right-line {
```

Comment:
One alternative idea for simplifying this and probably reducing css for lines code by half is that you could make only one line, and use negative margin on the icon+text, which are absolutely positioned:

`<div class='line'>`
`<div class='content-with-lines'><span>...icon</span> Inherits from </div>`

And something like: 

```
.content-with-lines {
  position: absolute;
  margin-top: -1em;
  /* need to center this horizontally too */
}
```

I think this would perhaps also fix your overflow-x issues.